### PR TITLE
feat(content): Revert missile changes + miscellaneous balance tweaks

### PR DESCRIPTION
**Pull request by Anarchist2**
## Summary did you know that the name of PR's branch is gullible
This is a simplified revision of #8622.
* Revert missile drag, acceleration and velocity values back to their pre-#6260 numbers. While missile dodging is easier now, these changes have also made missiles impossible to use by players at ranges longer than point-blank. Damage numbers and outfit space are not changed by this PR.
* ~Double radar jamming strength of the human jammers because they smell.~
* Reduce the Javelin Mini Pod's ammo count from 40 to 30. Javelins are by far and away the best missile type, and part of that is because of how spammable they are. This change should make Javelin Mini Pods less of the best weapon choice at that size.
* Increase the Sidewinder Pod's ammo from 4 to 6, and the Typhoon Pod's ammo from 2 to 3. Despite being larger than the Torpedo Pod, the Typhoon Pod carries less total damage (2100 vs 2460). While Sidewinders do deal less damage than Meteors, the Sidewinder Launcher holds more damage than the Meteor Launcher (9900 vs 8700), whereas the Sidewinder Pod holds less than half of the Meteor Pod (880 to 2030). These changes are so that those pods don't desperately need ammo racks to be usable.
* Altered ammo counts on some ships to account for these changes.

### CODE REVIEW

File: data/human/outfits.txt [hunk 1]
```diff
@@ -139,7 +139,7 @@ outfit "Large Radar Jammer"
 	thumbnail "outfit/large radar jammer"
 	"mass" 14
 	"outfit space" -14
-	"radar jamming" 10
+	"radar jamming" 20
```
Comment (OcelotWalrus) [inline data/human/outfits.txt:142]: Maybe doubling would be a bit too much? Why not values like 15?

File: data/human/ships.txt [hunk 2]
```diff
@@ -1373,7 +1373,7 @@ ship "Cutthroat"
 		"Twin Modified Blaster" 2
 		"Torpedo Pod" 2
 		"Torpedo Storage Rack"
-		"Torpedo" 21
+		"Torpedo" 23
```
Comment (Quantumshark) [inline data/human/ships.txt:1376]: You haven't changed the capacity of torpedo pods, so this doesn't need to be changed.

**Review by Quantumshark — APPROVED** [review]

**Comment by Zitchas** [comment]
I very much think that these changes should be broken up and done individually. Every line of the summary should be a separate PR and discussed individually.

Radar jamming and javeline pod are pretty solid, the rest are iffy and deserve to be discussed individually on their own merits.

**Comment by Hurleveur** [comment]
The only missile that's impossible to use is the Meteor, I think, and that could be fixed by tweaking the tracking.

I personally like the way things like torpdoes currently function, they're very dangerous up close but otherwise dodgeable, if we're changing that I'd argue the damage should change with it, because right now taking a torp to the face is too impactfull, even, and forces me to stay at a distance from ships with torps.

The jamming changes are a very good idea, and addresses sidewinder currently being too good

**Review by OcelotWalrus — APPROVED** [review]
I like the changes, it really balance the outfits in a good way.

**Comment by xX-Dillinger-Xx** [comment]
I personally don't care about secondary missile type weapons much, but will make this comment to explain why.  
They are largely ineffective for the player to use unless you have a medium or larger sized fleet of missile equipped ships.
If tracking was improved across the board, then maybe they become more viable for solo and small fleet players. Now, it is a real pain when you are facing an NPC fleet of missile equipped ships . 
So, if anything the Jamming buff is great. 
And nerfing everything else helps the players that don't use them.  Seeing as they are only effective in fleets, players that do use them can just add a couple more ships to bring the overall DPS back up.

**Comment by MasterOfGrey** [comment]
I second the idea of separating out at least the jamming component into something else.
Current jamming behaviour isn't effective, and boosting the strength of it doesn't make it more so.

It's all well and good for it to have a small chance of jamming off course from the target ship and looping around until it re-locks, but when you've gotta compete with AM (which is intrinsically a strong option) - if the missile missing it's target and hitting another friendly ship while jammed is still a likely outcome, it's pretty worthless.

Some time ago VitalChip demo'd a jamming behaviour on the discord which I'd endorse: [Here we go, this was the demo'd behaviour: ⁠content-creating⁠](https://discord.com/channels/251118043411775489/266345072554016768/824329373015998504)

Ideally that^ + jammed missiles being unable to impact friendly ships other than their intended target = what is needed to make jammers worthwhile.

Alternatively, Azure proposed a whole larger rework of missile targeting and jamming:

> Really I think all missiles/ship detection should have a sort of target radius with two relations to the ship being targeted
> - Center of the target zone, tied to the target and offset if being jammed
> - Radius of the target zone, out from the center and based on sensor power
>
>Every frame or whatever the missile's actual spatial target point would be randomly chosen from inside that target zone. 
>Missiles would ideally have a good enough sensor power for the radius of the target zone to be roughly on the order of a ship's size (though providing an advantage to smaller ships) and the target zone center position should pretty much always match the target ship without jamming.

If that were seriously considered then I'd suggest Azure's approach + jammed missiles being unable to impact friendly ships other than their intended target.

--------------

In any event, with current jamming behaviour there's no point quibbling over the jamming strength, because it's fundamentally just not good or meaningful except for the minority of people tryna do this game in a solo, narrative-appropriate tech-level ship.
...and even then, it's eh.

So that probably doesn't belong in this cluster of changes.

**Comment by Anarchist2** [comment]
Jamming isn't forced to compete against AM in every area, though. The ships that are most vulnerable to Sidewinders, and benefit most from radar jammers are smaller ships that are unable to install turrets at all, or would rather use their hardpoints and weapon capacity offensively. In this situation, I find that haywire missiles combined with high maneuverability is generally comparable to shooting the missile down, and the only issue is how space inefficient jammers are compared to extra battery or more ammo.

I think it's an inevitability that AM will always outclass jamming in fleets and on larger ships; they just play into each other's strengths and weaknesses better. At the same time, I don't think that jamming being limited to the earlygame is a damning flaw that should make us completely neglect it; everyone starts in a single ship, after all.

**Comment by MasterOfGrey** [comment]
> Jamming isn't forced to compete against AM in every area

As a player, with a desire for any degree of longevity for a given ship loadout, or with >1 ship (which is many people), it does compete in the overwhelming majority of areas.

> I think it's an inevitability that AM will always outclass jamming [...] on larger ships

That actually is kind of the major problem with jamming.
Turret slots are highly limited. The bigger a ship gets, the *more* economical it should be to predominantly rely on jamming instead of AM, because outfit capacity spent on jammers enables you to bring more offensive weapons - which helps you end the fight faster, reducing the amount of time needing to be spent on countering missiles.
The fact that even on very big ships, AM always outclasses jamming is the fundamental, underlying, *problem* with jamming as it current stands.

> I don't think that jamming being limited to the earlygame is a damning flaw

I do, because it *should* be something that comes into its own in the late-game. Having a large ship able to bring a suite of different jammers and focus its hardpoints on weaponry to win toe-to-toe fights faster should be the *goal*.

> that should make us completely neglect it;

Agreed, we shouldn't neglect it, but we should first recognise what are the actual defining constraints on ship outfitting - and limited hardpoints is the biggest, most restrictive constraint of them all. 
Therefore, the bigger a ship gets relative to the number of hardpoints it has, the more attractive alternatives that don't use hardpoints should become.

You perceive jamming as an early game thing that people move beyond, and argue that it's fine that it's mostly early-game with limited late-game utility because it's bad.
I argue jamming *should* be the primary defence against missiles late-game if there were sensible mechanics for it relative to ship progression, and that the *problem* is that it's too bad for that to be the case.

This is why I don't think that just tacking on a strength boost to it here is meaningful.
This is why I argue for the above changes to jamming.
This is why I think it's a discussion that deserves its own space.

**Comment by Hurleveur** [comment]
At the very least the strength boost wont hurt, and should help players until they get their first ship. We can always start with that as that's an easy change, and then look into other suggestions?

**Comment by Anarchist2** [comment]
I've removed the jammer changes from this PR.

**Comment by Anarchist2** [comment]
It was discussed with the reviewers that the remaining changes in this PR were all related enough to overall missile balance that they did not deserve being split up into their own PRs. If you have any issues with the changes, say it here.

**Pull request merged**